### PR TITLE
#164046198: Setup HoundCi to Check for Style Guide Violations

### PR DESCRIPTION
**Pull request by cob04**
**Description**

This **PR** adds code style checks
- Checks if the code follows style guidelines on every **PR** using `HoundCi`
- Uses the Airbnb style guide
- Uses `ESLint` as a dependency

<hr>
**Type of change**

- [x] feature(non-breaking change which adds style guide enforcement).

<hr>

**How to test this manually**
- When you create a **PR** to this repo, check for comments on `GitHub` by `HoundCI`.
- If there are issues, `HoundCi` will comment on them to point them out.
        Make sure you have these files as defined in this **PR**
         -  `.hound.yml`
         - `.eslintrc.json`
         - `.eslintignore`

**Checklist:**
- [x] This follows all the guidelines for this project.

#### Related Stories

[#164046198](https://www.pivotaltracker.com/story/show/164046198)

### CODE REVIEW

**Review by RachelleMaina — APPROVED** [review]
LGTM

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+    }
+}
```
Comment (MandelaK) [inline .eslintrc.json]: Please add EOF here

Comment (cob04) [reply .eslintrc.json]: Thank you, it is done.

Comment (BrianSerem) [reply .eslintrc.json]: Please add `"esversion": 6,` in this file for the hound to sniff the version of ECMAScript we are using. Read [more here ](https://stackoverflow.com/questions/27441803/why-does-jshint-throw-a-warning-if-i-am-using-const)

Comment (cob04) [reply .eslintrc.json]: I believe it's covered by this line `"ecmaVersion": 2018`.  Also, let me point out we are using `ESLint` not `JSHint`

**Pull request merged**